### ISSUE TITLE
Termcap issue while inside an interactive terminal in docker

### ISSUE THREAD

**Issue by jrgilman** [bobthecow/psysh#485]
@GrahamCampbell recommended that this should be posted here rather than in laravel/tinker:

I'm not exactly familiar with this issue, but I was on the ##linux channel on freenode trying to figure out what was causing it and the people there recommended that I bring this up as an issue.

The problem (seems to) only happens when inside of a docker container (docker exec -it XXX /bin/bash). Here is a gif of the problem: https://imgur.com/a/OCZ86

**Comment by bobthecow** [comment]
I'm unable to reproduce it in a docker machine on my end. To help narrow it down a bit, can you paste the output of `\Psy\info()`? Additionally, do you have a Dockerfile you can share, or even what your base image is?

**Comment by jrgilman** [comment]
\Psy\info():
```
>>> \Psy\info()
=> [
     "PsySH version" => "v0.8.18",
     "PHP version" => "7.1.16",
     "default includes" => [],
     "require semicolons" => false,
     "error logging level" => 32767,
     "config file" => [
       "default config file" => null,
       "local config file" => null,
       "PSYSH_CONFIG env" => false,
     ],
     "updates" => [
       "update available" => true,
       "latest release version" => "v0.9.3",
       "update check interval" => "never",
       "update cache file" => "~/.config/psysh/update_check.json",
     ],
     "pcntl" => [
       "pcntl available" => false,
       "posix available" => true,
     ],
     "readline" => [
       "readline available" => true,
       "readline enabled" => true,
       "readline service" => "Psy\Readline\Libedit",
       "readline library" => "EditLine wrapper",
     ],
     "history" => [
       "history file" => "~/.config/psysh/psysh_history",
       "history size" => null,
       "erase duplicates" => null,
     ],
     "docs" => [
       "manual db file" => null,
       "sqlite available" => true,
     ],
     "autocomplete" => [
       "tab completion enabled" => true,
       "custom matchers" => [],
       "bracketed paste" => false,
     ],
   ]
```

Dockerfile:
```
FROM php:7.1-fpm

COPY php.ini /usr/local/etc/php/

RUN echo "UTC" > /etc/timezone

RUN apt-get update && apt-get install -y libxml2-dev \
                                         libcurl4-openssl-dev \
                                         libzip-dev \
                                         libmcrypt-dev \
                                         libgd-dev \
                                         cron

RUN docker-php-ext-install pdo_mysql \
                           mbstring \
                           xml \
                           curl \
                           zip \
                           mcrypt \
                           gd

WORKDIR /tmp
COPY env-check.sh /tmp
COPY envsubst /usr/local/sbin

RUN echo "* * * * * php /var/www/artisan schedule:run >> /dev/null 2>&1" >> temp_cron \
 && crontab temp_cron \
 && rm temp_cron

RUN yes | pecl install xdebug
COPY xdebug.ini /usr/local/etc/php/conf.d

WORKDIR /var/www
COPY --chown=www-data:www-data www /var/www

# This is a total clusterfuck, I will look into not making this so shitty later.
# I'd rather the root user run a watcher of the www-data user's php-fpm process.
CMD /tmp/env-check.sh \
 && chown -R www-data:www-data /var/www \
 && php-fpm
```

**Comment by bobthecow** [comment]
Hmm. Even with that Dockerfile, I'm unable to reproduce this. I'm not sure if something in `env-check.sh` or `envsubst` or your `php.ini` or `debug.ini` is messing with things, though, because I don't have access to any of those. 

Do you mind commenting out those lines and seeing if the issue persists?

**Comment by jrgilman** [comment]
Strange, I know the error still happens without those files because I've been dealing with this since I first started using docker + php. I'll try to see if maybe it's not just an issue for me with my php containers, but I've never seen it with anything else...

Note, the error doesn't happen if your terminal is 80 characters long (that's what I've had to do in order to avoid this error). Your terminal needs to be longer than 80 characters.
